angular/src/app/workflows/workflow-detail/workflow-detail.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { JsonEditorComponent, JsonEditorOptions } from 'ang-jsoneditor';
+
+import { JsonEditorDefaults } from '../../admin/dam-repository/shared/jsonEditorDefaults';
+import { WorkflowService } from '../workflows.service';
+
+@Component({
+  selector: 'ddap-workflow-detail',
+  templateUrl: './workflow-detail.component.html',
+  styleUrls: ['./workflow-detail.component.scss'],
+})
+export class WorkflowDetailComponent implements OnInit {
+  runDetails;
```

Comment:
This should be of type `WorkflowRun` from `workflow.model.ts`. You need to also update that model a little.